src/diffTool/diffProcessor.ts

```diff
@@ -12,20 +12,42 @@ import { ramlToolLogger } from "../common/logger";
 import * as path from "path";
 import { applyRules } from "./rulesProcessor";
 
-export const diffRulesPath = path.join(__dirname, "../../resources/diff/rules");
+export const defaultRulesPath = path.join(
+  __dirname,
+  "../../resources/diff/rules/defaultRules.json"
+);
+
+/**
+ * Generate differences between two RAML files and apply rules
+ * @param leftRaml - Base RAML file to compare
+ * @param rightRaml - Other RAML file to compare with the left RAML to find differences
+ * @param rulesPath - Optional rules file to apply on the differences
+ *
+ * @returns Changes to the APIs
+ */
+export async function findApiChanges(
```

Comment:
Should this be `applyChangeRules` instead?

Reply:
There is already function apply rules that applies the rules on node diffs. 
`applyRules(diffs: NodeDiff[],rulesPath: string)`

I thought we agreed on "findApiChanges". Once we have **ApiChanges** object, its return value will also be changed to **ApiChanges**

Reply:
you're right.  I retract my comment